README.md

```diff
@@ -11,6 +11,27 @@ PyMC-Marketing is an open source Python package for Bayesian marketing analytics
 
 ---
 
+## Installation
+
+You may have already set up an environment (e.g. `marketing_test_env`) you’d like to work in. It may look something like the following:
+
+```
+mamba create -n marketing_test_env python=3.10 matplotlib jupyterlab ipykernel seaborn pandas
```

Comment:
shouldn't you add pymc here?

Reply:
it's a dependency of pymc-marketing, so it's installed automatically. But we could if you'd prefer

Reply:
`pymc`, and through `conda-forge` channel.

My original suggestion was to just direct people to the PyMC installation followed by pip install pymc-marketing

Reply:
I'll edit this a bit